sql/migrations/035_jira_user.up.sql

```diff
@@ -0,0 +1,34 @@
+BEGIN;
+
+ALTER TABLE jira.issue ADD COLUMN reporter_id text;
+UPDATE jira.issue SET reporter_id = 'needs re-fetch';
+ALTER TABLE jira.issue ALTER COLUMN reporter_id SET NOT NULL;
+
+ALTER TABLE jira.issue ADD COLUMN reporter_display_name text;
+UPDATE jira.issue SET reporter_display_name = 'needs re-fetch';
+ALTER TABLE jira.issue ALTER COLUMN reporter_display_name SET NOT NULL;
+
+ALTER TABLE jira.issue ADD COLUMN assignee_id text;
+ALTER TABLE jira.issue ADD COLUMN assignee_display_name text;
+
+-- TODO: commenters IDs
+ALTER TABLE jira.issue ADD COLUMN commenters_display_names text[];
+
+-- TODO: priority ID
+ALTER TABLE jira.issue ADD COLUMN priority_name text;
+UPDATE jira.issue SET priority_name = 'needs re-fetch';
+ALTER TABLE jira.issue ALTER COLUMN priority_name SET NOT NULL;
+
+CREATE INDEX IF NOT EXISTS issue_reporter ON jira.issue(acc_id, created, resolved, lower(priority_name), lower(reporter_display_name));
+CREATE INDEX IF NOT EXISTS issue_resolver ON jira.issue(acc_id, created, resolved, lower(priority_name), lower(assignee_display_name));
```

Comment:
Having spent a few hours coding sample SQL, these seem to be needed:
```suggestion
CREATE INDEX IF NOT EXISTS issue_reporter ON jira.issue(acc_id, created, resolved, lower(reporter_display_name), lower(priority_name));
CREATE INDEX IF NOT EXISTS issue_resolver ON jira.issue(acc_id, created, resolved, lower(assignee_display_name), lower(priority_name));
CREATE INDEX IF NOT EXISTS issue_commenters ON jira.issue USING GIN(acc_id, created, resolved, commenters_display_names, lower(priority_name));
CREATE INDEX IF NOT EXISTS issue_reporter_active ON jira.issue(acc_id, updated, created, resolved,  lower(reporter_display_name), lower(priority_name));
CREATE INDEX IF NOT EXISTS issue_resolver_active ON jira.issue(acc_id, updated, created, resolved, lower(assignee_display_name), lower(priority_name));
CREATE INDEX IF NOT EXISTS issue_commenters_updated ON jira.issue USING GIN(acc_id, updated, created, resolved, commenters_display_names, lower(priority_name));
```
IDK much about GIN, we really need Denys to adjust here... If it does not work with GIN then we can add it later.

Reply:
agreed to skip for now